src/main/java/net/dv8tion/jda/internal/entities/ApplicationInfoImpl.java

```diff
@@ -40,10 +43,14 @@
     private final String name;
     private final User owner;
     private final ApplicationTeam team;
+    private final List<String> tags;
+    private final String customAuthUrl;
+    private final EnumSet<Permission> defaultAuthUrlPerms;
```

Comment:
These should be stored as `long`, there isn't really a reason to parse them unless the user needs it.